js/slouchdetection.js

```diff
@@ -4,6 +4,11 @@
 
 const LOWERED_FACE_DISTANCE = 30;
 const TOO_BIG_FACE_SIZE = 200;
+const SLOUCHOPTIONS = {
+    "forwardLeaningSlouch": 1,
+    "slouchDown": 2,
+    "none": false
+};
```

Comment:
Use `CAPITALS_WITH_UNDERSCORES` to name enum values and use 0, 1, and 2 as values